src/Shared/Components/header/Breadcrumb.js

```diff
@@ -64,8 +69,18 @@ const Breadcrumb = ({ partLabel }) => {
 
   // Actual BreadcrumbBar component. If the route name is Home, do not display breadcrumb.
   return (
-    <View style={styles.breadcrumb}>
-      {route.name === "Home" ? <BlankBar /> : <NavigationLink />}
+    <View>
+      <View style={styles.breadcrumb}>
+        {route.name === "Home" ? <BlankBar /> : <NavigationLink />}
+        {showIcon ? (
```

Comment:
This should be moved into the NavigationLink

Reply:
Okay I think it sort of works now.

Reply:
It totally works now.